Translated/Dialog/Nandel/ssn_014250.csv

```diff
@@ -0,0 +1,11 @@
+ssn_014250_0001#0,"""I'm Nandel. I'm in charge of guarding the Southern Gate."""
+ssn_014250_0002#0,"""I might be the youngest out of all of the cardinal gateskeepers, but I've got the most job experience."""
+ssn_014250_0003#0,"""I work extra hard to be someone that my colleagues can look up to."""
+ssn_014250_0101#0,"""Your name has certainly been buzzing around the rumour-mill. You've been hard at work!"""
```

Comment:
We stick to US spelling as standard. Rumor. You used GB spelling in a couple other places too.

Reply:
...with all due respect, i didn't use US spelling because i'm not an american. is that really a mistake that needs fixing 😐

Reply:
Neither am I, but it's a standard we should follow across the board.

Reply:
i literally don't know all of the words that have an american spelling so i don't know how i can be expected to follow it. but alright.

Reply:
fixed. 🤠